### PR TITLE
[ADF-5022] added automation for Display Value Widget

### PR DESCRIPTION
**Pull request by AleAb25**
**Please check if the PR fulfills these requirements**

> - [ x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ x] Other... Please describe: automating regression tests

**What is the current behaviour?** (You can also link to an open issue here)
https://issues.alfresco.com/jira/browse/ADF-5022

**What is the new behaviour?**
Automation is added for Display Value Widget - visibility-conditions.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: e2e/process-services-cloud/form-field/visibility-condition.e2e.ts [hunk 1]
```diff
@@ -242,4 +250,79 @@ describe('Visibility conditions - cloud', () => {
         await widget.textWidget().setValue(widgets.textThreeId, 'something');
         await widget.textWidget().isWidgetVisible(widgets.textOneId);
     });
+
+    it('[C309867] Display value widget displays the value of a form variable', async () => {
```
Comment (DenysVuika) [inline e2e/process-services-cloud/form-field/visibility-condition.e2e.ts]: Incorrect test name format

Comment (AleAb25) [reply e2e/process-services-cloud/form-field/visibility-condition.e2e.ts]: Corrected the name & updated in TestRail as well.

File: lib/testing/src/lib/core/pages/form/widgets/displayValueWidget.ts [hunk 2]
```diff
@@ -16,7 +16,8 @@
  */
 
 import { FormFields } from '../formFields';
-import { by, Locator } from 'protractor';
+import { by, element, Locator } from 'protractor';
+import { BrowserVisibility } from '@alfresco/adf-testing';
```
Comment (eromano) [inline lib/testing/src/lib/core/pages/form/widgets/displayValueWidget.ts]: this import create a circular dependency please see the other file inside this module how they import the BrowserVisibility.

File: lib/testing/src/lib/core/pages/form/widgets/displayValueWidget.ts [hunk 3]
```diff
@@ -32,4 +33,12 @@ export class DisplayValueWidget {
         return this.formFields.getFieldValue(fieldId, this.inputLocator);
     }
 
+    async isDisplayValueWidgetVisible(fieldId): Promise<void> {
```
Comment (eromano) [inline lib/testing/src/lib/core/pages/form/widgets/displayValueWidget.ts]: missing type for fieldId in both methods of this page

File: lib/testing/src/lib/core/pages/form/widgets/displayValueWidget.ts (hunk 2 above)
Comment (AleAb25) [reply lib/testing/src/lib/core/pages/form/widgets/displayValueWidget.ts]: changed the import line for BrowserVisibility

File: lib/testing/src/lib/core/pages/form/widgets/displayValueWidget.ts (hunk 3 above)
Comment (AleAb25) [reply lib/testing/src/lib/core/pages/form/widgets/displayValueWidget.ts]: added type for fieldId

**Pull request merged**